packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -97,13 +97,13 @@ import {
   useFocus, useFocusEmits,
   useStateful, useStatefulEmits, useStatefulProps,
   useTranslation,
-  useDropdownable, useDropdownableProps, useDropdownableEmits,
+  useDropdownable, useDropdownableProps, useDropdownableEmits, useLongPressKey,
 } from '../../composables'
 import { useTimeParser } from './hooks/time-text-parser'
 import { useTimeFormatter } from './hooks/time-text-formatter'
 
```

Comment:
max-width for default time input 85px 
![image](https://github.com/epicmaxco/vuestic-ui/assets/35342240/c77792df-1792-401f-bdbf-4c7953d93e5c)

Reply:
I'd better keep time input width 120px (the same we have for Counter, DateInput, ColorInput). There are might be seconds added. This way we also keep consistence, so there are large inputs (250px) and small inputs (120px)

Reply:
Shouldn't input be full-width by default?

Or we're speaking about examples here?

Reply:
Great question. I think it is something we need to discuss. 

Actually, inputs by default have some width (250px), which can be changed with style="width: 400px" or in case it is in flex container input will grow in side to take 100%. This behavior is similar to native input. So we do have some default width. 

But having different default width cause form which consists of different type of inputs to look wacky. Nevertheless, having small input size for date-input makes it look good for single mode, but in case we select range of dates the text content simply doesn't fit. (same with time picker if we add seconds or period, color if input is in rgba, etc).

This is why I think we should have one default input size in any kind of inputs. 

In demo though, we can manually set width to make inputs prettier, but I don't like it neither. Usually, when I see a default usage example, I'm not really going into reading code example. I just use `<VaDateInput />` in my app. I'll be confused to see different size in my app and example.

I tried to make different size in docs and it confused me as docs user.

Reply:
I checked a couple of solutions in diferent frameworks, and it seems, that modern frameworks use full width.

Element-ui, on the other hand, uses different sizes to different input types, but overall, there is not too many sizes (input is 100%, select/autocomplete is about 220, date pickers and count input are perfect fits for content).

So I'm thinking maybe we should use full width for all inputs, so that it's maybe sometimes more work, but very consistent.

Would be nice to have a more solid evidence though, that's a big change and we probably won't be able to change this ever again.

Reply:
Asked ChatGPT. They think it's 100% + customization.

100% Width vs Specific Sizes for Form Elements
Eugene: I'm working on UI library on form elements. I'm thinking whether to have them all be 100% width or have specific sizes for every one of them. Can you think of cons and pros for each claim?

ChatGPT:

100% Width for All Elements
Pros:
Consistency: All elements will have a unified look.
Responsiveness: Ensures adaptation to different screen sizes.
Simpler Development: Easier development and maintenance.
Cons:
Lack of Flexibility: Limits customization.
Potential Design Issues: May look awkward at full width.
Specific Sizes for Each Element
Pros:
Customized Appearance: Tailors appearance to function.
Control: More control for finely-tuned design.
Avoiding Overstretching: Prevents elements from looking bad when stretched.
Cons:
Increased Complexity: Makes development more complex.
Potential Responsiveness Issues: Can cause issues across devices.
Consistency Challenges: Extra effort to ensure a cohesive look.
Eugene: If you would make a choice - what would that be?

ChatGPT: I recommend the 100% width approach, but with provisions for customization. This aligns with modern responsive design principles, simplifies initial development, and provides optional customization to cater to specific needs. It combines the strengths of both strategies without overwhelming complexity.